### DISCUSSION TITLE
`RENOVATE_IGNORE_PR_AUTHOR` (`ignorePrAuthor`) best practice for GitLab self-hosted

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [renovatebot/renovate#18876] [Request Help]
From the documentation it is suggested to have `RENOVATE_IGNORE_PR_AUTHOR = false`. I see no reasons to use `true` for the GitLab self-hosted renovate-bot.

At the same time, [GitLab default renovate-bot config](https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/templates/_common.gitlab-ci.yml#L10) has `RENOVATE_IGNORE_PR_AUTHOR: 'true'`. 

Why the default is `true` for GitLab self-hosted? I would like to use `false` based on the documentation suggestions, but I am afraid that GitLab renovate-bot template maintainer had some good reasons to change this value.

**Comment by rarkins** [comment]
@viceice was this possibly for backwards compatibility when existing users switched from using `gitlab.com/renovate-bot` to their own user? And if so then new users can set it to false?

**Reply by viceice** [reply]
yes, i think so
